.gitignore

```diff
@@ -18,3 +18,7 @@ coverage
 
 # sphinx / readthedocs
 _build
+
+
+# IDES
+.vscode
```

Comment:
pp: does not hurt but I would encourage you to setup your individual config in a global `.gitignore` file in your home directory. You may find this helpful: https://github.com/alpe/dotfiles/blob/master/.gitignore

Reply:
IMHO it's actually quite nice to `.gitignore` the popular IDEs as it's helpful for any other contributors too. The less personalised setup you need to contribute to things - the better!

Reply:
As an interesting anegdote, I worked once in a company where in the application repository there was a docker compose configuration that was bootstrapping the application together **with visual studio** so that everyone "could" use the same setup :speak_no_evil::finnadie:

Reply:
ha! had the same with some java projects. Didn't really work out of the box though, I remember it took me a couple of days to get this properly running and that's with a "set up" project.